doc/source/computation.rst

```diff
@@ -520,11 +525,16 @@ For example:
 Computing rolling pairwise covariances and correlations
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. warning::
+
+   Prior to version 0.20.0 if ``pairwise=True`` was passed, a ``Panel`` would be returned.
```

Comment:
What confused me here is that you don't need to pass this `pairwise=True`, as it is the default when doing a rolling corr with self. 
But it is correct it is only with pairwise? So I would maybe first just say something like "when pairwise rolling correlation is calculated, a Panel ..." and only after say like "(the default when no other frame is provided or when `pairwise=True`")